tools/mti-utils/rename_tiff_channels.xml

```diff
@@ -93,12 +93,13 @@ print(tiffcomment(os.path.join(cwd, 'renamed_image.ome.tiff')))
             <param name="channel_csv" value="rename_channels.csv" />
             <output name="renamed_image" ftype="ome.tiff">
                 <assert_contents>
-                    <has_size value="899000" delta="50000" />
+                    <has_size value="900000" delta="50000" />
```

Comment:
can you already use the new image asserts for testing? With this test should be much more stable...

Reply:
Not quite yet unfortunately. We have a few priorities we'd like to finish up before we go about updating our instances to Galaxy version 24, and in the meantime, we'd like our CI to reflect the version we are running on our instances. I've created an issue so that I remember to come back through and put in more robust image assertions once we get the chance to update: https://github.com/goeckslab/tools-mti/issues/72 

I'll add this tool, and other tools that would benefit from the new image assertions to the issue.

Thank you very much for your feedback and help!